ts/navigation/AppStackNavigator.tsx

```diff
@@ -93,6 +96,12 @@ const InnerNavigationContainer = (props: InnerNavigationContainerProps) => {
     isNewPaymentSectionEnabledSelector
   );
 
+  // ITW Flags
+  const isItwTrialActive = useIOSelector(isItwTrialActiveSelector);
+  const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
+  const isItwEnabled = useIOSelector(isItwEnabledSelector);
+  const canBeActivated = isItwTrialActive && !isItwValid && isItwEnabled;
```

Comment:
```suggestion
  const canItwBeActivated = isItwTrialActive && !isItwValid && isItwEnabled;
```

I think `canBeActivated` is too generic and the constant name should be more explicit, especially when used later.

Reply:
Done, thanks.